index.html

```diff
@@ -91,7 +91,8 @@
             <div class="ui-block-b"><button type="submit" data-theme="a" id="b-submit">Submit</button></div>
           </fieldset>
 	  <br>
-	  <p>OpenHAB for Pebble by Richard Lee</p>
+	  <p>openHAB Pebble application by Richard Lee</p>
+	  <p>openHAB software by openhHAB Foundation <i>http://www.openhabfoundation.org/<i></p>
```

Comment:
typo in "openhHAB" - but anyhow, you can leave out this line. The foundation is not doing the software, it is only helping to make it accessible.